### ISSUE TITLE
[@pika/plugin-standard-pkg] Plugin appears to ignore .babelrc, Fails to transform .gql files

### ISSUE THREAD

**Issue by Saeris** [pikapkg/builders#116]
I can't get my package to build using `@pika/plugin-standard-pkg`, as it throws a syntax error trying to read my `.gql` file imports, which should be transformed via `babel-plugin-import-graphql` at build time, which would replace the file imports with inline'd calls to `graphql-tag` instead. Here's what's coming up in my console:

```bash
$ yarn build
yarn run v1.21.1
$ cross-env PIKA=true pika build
@pika/pack v0.5.0
[1/6] Validating source...
[2/6] Preparing pipeline...
      ❇️  pkg\
[3/6] Running @pika/plugin-standard-pkg...
babelrc loaded! { PIKA: 'true' }
SyntaxError: D:\GitHub\@saeris\mini-movie-db-api\src\types\backdrop.gql: Unexpected token, expected ";" (1:5)

> 1 | type Backdrop implements Image {
    |      ^
  2 |   file: String
  3 |   custom(size: String svg: Boolean color: Palette base64: Boolean): URL
  4 |   original(svg: Boolean color: Palette base64: Boolean): URL
    at Parser._raise (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:723:17)
    at Parser.raiseWithData (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:716:17)
    at Parser.raise (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:710:17)
    at Parser.unexpected (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:8606:16)
    at Parser.semicolon (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:8588:40)
    at Parser.parseExpressionStatement (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:11415:10)
    at Parser.parseStatementContent (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:11016:19)
    at Parser.parseStatement (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:10882:17)
    at Parser.parseBlockOrModuleBlockBody (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:11456:25)
    at Parser.parseBlockBody (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@babel\parser\lib\index.js:11443:10)
Error: Command failed with exit code 1 (Unknown system error -1): npx pika-pack
    at makeError (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@pika\cli\node_modules\execa\lib\error.js:59:11)
    at handlePromise (D:\GitHub\@saeris\mini-movie-db-api\node_modules\@pika\cli\node_modules\execa\index.js:112:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Spent quite a while trying to figure out if I had just configured something wrong, but I can't seem to find an answer. My first guess was that `standard-pkg` was not reading my `.babelrc.js` file, but adding a console log to it quickly showed that it was indeed being read (which you can see in the output above right before the error).

I'll admit this is definitely an edge case, but I'd love to find a solution to this problem. You can check out the [repo](https://github.com/Saeris/mini-movie-db-api) to reproduce the error and help debug. `yarn build` should be all you need to run.

**Comment by FredKSchott** [comment]
Hey! We actually don't do anything special here, and rely on the Babel package's internal resolution to resolve these for us. If I have some time I can dig into this with you, but unfortunately there's nothing going on on our end that I'm aware of that could be causing this.

**Comment by Saeris** [comment]
Found a solution!

Looks like I had the wrong glob pattern in my exclude array. Since `standard-pkg` will loop through every file in your source and attempt to transform it, if you don't explicitly exclude non-JS files, it will try to parse them as JS, as it was doing here, resulting in the syntax error I encountered. Here's an example of how to exclude a file extension properly:

```javascript
"@pika/pack": {
  "pipeline": [
    [
      "@pika/plugin-standard-pkg",
      {
        "exclude": [
          "**/*.gql"
        ]
      }
    ]
}
```

Previously I had `*.gql` instead of `**/*.gql`, which was the source of the problem.

Now on to figure out why the TypeScript step is failing...
